index.js

```diff
@@ -84,8 +90,9 @@ function commist() {
   }
 }
 
-function Command(string, func) {
+function Command(string, strict, func) {
```

Comment:
I think the inputs here could become an option object

Reply:
Done 👍